mycroft/audio/services/__init__.py

```diff
@@ -15,7 +15,7 @@
 from abc import ABCMeta, abstractmethod
 
 
-class AudioBackend():
+class AudioBackend(object):
```

Comment:
No need to inherit from object in python3

Reply:
isn't that the standard however? we are doing it everywhere else

Reply:
Yeah I've noticed it was still hanging around. Added a PR to remove them.

Reply:
see https://github.com/MycroftAI/mycroft-core/pull/1945#issuecomment-453237439

either way i do not oppose this change, when #1945 is merged i'll update this